cf_xarray/tests/test_geometry.py

```diff
@@ -126,6 +126,154 @@ def geometry_line_without_multilines_ds():
     return cf_ds, shp_da
 
 
+@pytest.fixture
+def geometry_polygon_without_holes_ds():
+    from shapely.geometry import Polygon
+
+    # empty/fill workaround to avoid numpy deprecation(warning) due to the array interface of shapely geometries.
+    geoms = np.empty(2, dtype=object)
+    geoms[:] = [
+        Polygon(([50, 0], [40, 15], [30, 0])),
+        Polygon(([70, 50], [60, 65], [50, 50])),
+    ]
+
+    ds = xr.Dataset()
+    shp_da = xr.DataArray(geoms, dims=("index",), name="geometry")
+
+    cf_ds = ds.assign(
+        x=xr.DataArray(
+            [50, 40, 30, 50, 70, 60, 50, 70], dims=("node",), attrs={"axis": "X"}
+        ),
+        y=xr.DataArray(
+            [0, 15, 0, 0, 50, 65, 50, 50], dims=("node",), attrs={"axis": "Y"}
+        ),
+        node_count=xr.DataArray([4, 4], dims=("index",)),
+        crd_x=xr.DataArray([50, 70], dims=("index",), attrs={"nodes": "x"}),
+        crd_y=xr.DataArray([0, 50], dims=("index",), attrs={"nodes": "y"}),
+        geometry_container=xr.DataArray(
+            attrs={
+                "geometry_type": "polygon",
+                "node_count": "node_count",
+                "node_coordinates": "x y",
+                "coordinates": "crd_x crd_y",
+            }
+        ),
+    )
+
+    cf_ds = cf_ds.set_coords(["x", "y", "crd_x", "crd_y"])
+
+    return cf_ds, shp_da
+
+
+@pytest.fixture
+def geometry_polygon_without_multipolygons_ds():
+    from shapely.geometry import Polygon
+
+    # empty/fill workaround to avoid numpy deprecation(warning) due to the array interface of shapely geometries.
+    geoms = np.empty(2, dtype=object)
+    geoms[:] = [
+        Polygon(([50, 0], [40, 15], [30, 0])),
+        Polygon(
+            ([70, 50], [60, 65], [50, 50]),
+            [
+                ([55, 55], [60, 60], [75, 55]),
```

Comment:
```suggestion
                ([55, 55], [60, 60], [65, 55]),
```
I'm guessing a typo. The polygon is otherwise invalid!
Old in red, new in blue.
![image](https://github.com/xarray-contrib/cf-xarray/assets/20629530/fd571da4-f324-4c2c-b1b0-1e4ec6cb98ad)

Reply:
Yes that is definitely a typo. Thanks or the visual! I guess if shapely doesn't complain about invalid shapes neither should we?